src/main/java/staffconnect/model/person/comparators/PhoneComparator.java

```diff
@@ -15,4 +15,9 @@ public class PhoneComparator implements Comparator<Person> {
     public int compare(Person p1, Person p2) {
         return p1.getPhone().toString().compareToIgnoreCase(p2.getPhone().toString());
     }
+
+    @Override
+    public String toString() {
+        return "Phone by alphabetical order";
```

Comment:
Should `alphabetical order` be used for `Phone`?
_Might need to update the image if this is edited._

Reply:
will ammend!